formats/financial_release/frontend/examples/financial_release.json

```diff
@@ -0,0 +1,14 @@
+{
+  "base_path": "/lloyds-share-offer",
+  "content_id": "74314796-eee4-4a66-bc1e-3a2039f680e6",
+  "title": "Financial Release",
+  "format": "financial_release",
+  "locale": "en",
+  "public_updated_at": "2015-12-01T00:00:00.000Z",
+  "details": {
+    "summary": "Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,quis nostrud exercitation",
+    "published_date": "2015-10-05T00:00:00.000Z",
```

Comment:
I think `first_published_at` would be better in that case, as there's prior art for that:

https://github.com/alphagov/govuk-content-schemas/blob/4e1a2abc22afc65f3f1b887222103448294bdd79/dist/formats/specialist_document/publisher_v2/schema.json#L614-L617